packages/esm-patient-list-app/src/index.ts

```diff
@@ -25,6 +26,15 @@ function setupOpenMRS() {
   const route = `patient-list`;
   const spaBasePath = `${window.spaBase}/${route}`;
 
+  async function getName(x: string) {
+    try {
+      const name = await getPatientListName(x);
+      return name;
+    } catch (error) {
+      return error.message;
+    }
+  }
```

Comment:
```suggestion
  async function getListName(patientListUuid: string): Promise<string> {
    return (await getPatientListName(patientListUuid)) ?? '--';
  }
```

Reply:
Error handling has already been catered for in `getPatientListName`, so here we return either the resolved patient list name or `--` as a fallback.